cpp/src/arrow/filesystem/gcsfs_test.cc

```diff
@@ -797,6 +838,11 @@ TEST_F(GcsIntegrationTest, OpenInputStreamInfoInvalid) {
   ASSERT_RAISES(IOError, fs->OpenInputStream(info));
 }
 
+TEST_F(GcsIntegrationTest, OpenInputStreamUri) {
+  auto fs = internal::MakeGcsFileSystemForTest(TestGcsOptions());
+  ASSERT_RAISES(Invalid, fs->OpenInputStream("gcs:" + PreexistingObjectPath()));
```

Comment:
I know this is just for testing purposes, but the conventional prefix for GCS is `gs://`, at least that is what `gsutil` uses:

https://cloud.google.com/storage/docs/gsutil#syntax

Reply:
I see, thank you.
By the way, it seems other packages have adopted the `gcs://` convention: https://gcsfs.readthedocs.io/en/latest/#integration

Reply:
Sigh... My *guess* (and let me stress that this would be just a guess) is that `gs://` is more familiar to GCS users.  I can live with `gcs://` too

Reply:
Ah, it looks like they accept `gs://` too: https://github.com/fsspec/gcsfs/blob/main/gcsfs/core.py#L1171